app/autoloader.js

```diff
@@ -0,0 +1,5 @@
+function register(name, value) {
+    global.__defineGetter__(name, () => value);
```

Comment:
`__defineGetter__` is non-standard AND deprecated:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/__defineGetter__

We need to find an alternative.